src/explore-education-statistics-admin/src/pages/release/data/__tests__/ReleaseApiDataSetVersionHistoryPage.test.tsx

```diff
@@ -40,7 +40,7 @@ describe('ReleaseApiDataSetVersionHistoryPage', () => {
       },
       published: '2024-04-01',
       releaseVersion: {
-        id: 'release-version-3-1-id',
+        id: 'release-3-1-id',
```

Comment:
Out of interest, why remove the "version" part from the id?  I thought that we were trying to move towards making the distinction between Releases and ReleaseVersions more apparent in the front end code.  I think Ben in particular is keen that we move in this direction.

Reply:
This is just for brevity as including 'version' creates longer URLs that we have to assert on later. These already don't wrap well, so just to make things more readable, I've shortened them down as it's only test data and doesn't actually impact functionality.

Reply:
No worries, thanks